src/applications/vaos/services/appointment/transformers.js

```diff
@@ -236,7 +235,7 @@ export function transformVAOSAppointment(appt) {
       isInPersonVisit,
       isVideoAtHome,
       isVideoAtVA,
-      isCerner,
+      isCerner: appt.isCerner || false,
```

Comment:
Let me know if we should hold off on this change until https://github.com/department-of-veterans-affairs/vets-api/pull/23809 is deployed since this could impact staging validation if merged before the vets-api changes are live. Alternatively, we can split this into a separate PR if we want to unblock the Sentry logs first.

Separately, do we know that staging data has the correct identifier/system values? I checked a few for Judy and they seemed like they would be correct but @simiadebowale @ryanshaw would you know for sure?